heroic-test/src/main/java/com/spotify/heroic/test/AbstractSuggestBackendIT.java

```diff
@@ -146,6 +150,17 @@ public final void abstractTeardown() throws Exception {
         core.shutdown().get();
     }
 
+    @Test
+    public void writeDuplicatesReturnErrorInResponse() throws Exception {
```

Comment:
Since this is only relevant to SuggestBackendKVIT, it should be moved into that class instead of overriding it with a blank test in MemoryBackendIT.

Reply:
I tried moving it there but ran into timing issues brining up the elasticsearch test container. let me see if I can work past that...